sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/DocumentLine.java

```diff
@@ -85,10 +88,33 @@ public List<DocumentSpan> getSpans() {
      * @param spans the spans value to set.
      * @return the DocumentLine object itself.
      */
-    void setSpans(List<DocumentSpan> spans) {
+    private void setSpans(List<DocumentSpan> spans) {
         this.spans = spans;
     }
 
+    private void setPageWords(List<DocumentWord> pageWords) {
+        this.pageWords = pageWords;
+    }
+
+    /**
+     * Get the words found in the spans of this DocumentLine.
+     *
+     * @return the list of {@link DocumentWord} in this DocumentLine
+     */
+    public List<DocumentWord> getWords() {
```

Comment:
Maybe we leave that for post-GA, but would it make sense to do a binary search since spans are ordered?